tests/runtests.py

```diff
@@ -20,30 +20,8 @@
 
 from unittest import defaultTestLoader, TextTestRunner
 import sys
-import os
 import subprocess
```

Comment:
Please remove

Reply:
Done with a force push

Reply:
I'll let the tests return and then merge...

Reply:
The linter step failed to fetch the Python 3.8 tarball thus run:
```
Downloading archive: https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-3.8.tar.bz2
0.39s$ curl -sSf --retry 5 -o python-3.8.tar.bz2 ${archive_url}
curl: (56) GnuTLS recv error (-54): Error in the pull function.
Unable to download 3.8 archive. The archive may not exist. Please consider a different version.
```

Reply:
Probably a network error. Let's re-run...